scripts/readmeFiles.js

```diff
@@ -0,0 +1,189 @@
+${description}
+
+## Installation
+
+\`\`\`
+yarn add ${name}
+\`\`\`
+
+or with npm:
+
+\`\`\`
+npm install ${name}
+\`\`\`
+
+## Props 
+${props}
+
+## Usage
+<!-- autogenerated don't modify -->
+${content}
+
+#### more:
+- [Storybook showcase](https://paprika.highbond.com/?path=/story/${name.split("/")[1].toLowerCase()}--showcase)
+- [Github source code](https://github.com/acl-services/paprika/tree/master/packages/${displayName}/src)
+- [Github create issue](https://github.com/acl-services/paprika/issues/new?label=[]&title=${name}%20[help]:%20your%20short%20description&body=${githubIssueBody})
+- [ChangeLog](https://github.com/acl-services/paprika/tree/master/packages/${displayName}/CHANGELOG.md)
+`;
+};
+
+const initAutoReadme = ({ path, content }) => {
+  fs.writeFileSync(
+    `${path}/README.md`,
+    `<!-- content -->
+${content}
+<!-- eoContent -->
+      `,
+    "utf8"
+  );
+};
+
+const createPropsTable = ({ info }) => {
+  if (!info.props) return "";
+
+  const table = [
+    `### ${info.displayName} \n`,
+    "| Prop  | Type  | required  | default   | Description |\n",
+    "|-------|-------| --------  | --------- | ----------- |\n",
+  ];
+
+  Object.keys(info.props).map(key => {
+    const v = info.props[key] || {};
+    let type = "-";
+    if ("type" in v) {
+      type = v.type.name === "enum" ? v.type.value : v.type.name;
+    }
+
+    const required = "required" in v ? v.required.toString() : " ";
+    const defaultValue = "defaultValue" in v ? v.defaultValue.value : "-";
+    const description = "description" in v ? v.description : " ";
+
+    return table.push(`|${key}|${type}|${required}|${defaultValue}| ${description}|\n`);
+  });
+
+  return table.join("");
+};
+
+const extractCorrectComponentDefinition = ({ desireDefinition, arrayOfComponentsDefinitions }) => {
+  const definition = arrayOfComponentsDefinitions.filter(def => def.displayName === desireDefinition);
+  return definition[0];
+};
+
+const processPropTables = ({ info, folder, path, paprikaDocs = null }) => {
+  console.log("processing", folder);
+  const propsTable = [];
+  // const autoGeneratedContent = readmeTemplate({ name, description, content: contentExtracted });
+
+  propsTable.push(createPropsTable({ info }));
+
+  // you can define on packages.json a property named paprikaDocs with an attribute subComponent to list
+  // extra component that you might want to create and render on the table.
+  if (paprikaDocs && "subComponents" in paprikaDocs) {
+    paprikaDocs.subComponents.map(subComponent => {
+      const subComponentContent = fs.readFileSync(`${path}/src/components/${subComponent}/${subComponent}.js`, "utf8");
+      const arrayOfComponentsDefinitions = reactDocs.parse(
+        subComponentContent,
+        reactDocs.resolver.findAllComponentDefinitions
+      );
+      let _info = extractCorrectComponentDefinition({
+        desireDefinition: subComponent,
+        arrayOfComponentsDefinitions,
+      });
+
+      if (_info) {
```

Comment:
this means that the subComponent hasn't being exported with a proper display name like `MyComponent.Subcomponent` so it can be found on the array of definition, but if the displayName it's correct then we need to search for the component on the array with the proper name displayName `MyComponent.Subcomponent`